IDE/ARDUINO/wolfssl-arduino.sh

```diff
@@ -86,60 +218,126 @@ EOF
 
 
 # Creates user_settings file if one does not exist
-    if [ ! -f ".${ROOT_SRC_DIR}/user_settings.h" ]; then
-    	cat > .${ROOT_SRC_DIR}/user_settings.h <<EOF
-/* Generated wolfSSL user_settings.h file for Arduino */
-#ifndef ARDUINO_USER_SETTINGS_H
-#define ARDUINO_USER_SETTINGS_H
-
-/* Platform */
-#define WOLFSSL_ARDUINO
-
-/* Math library (remove this to use normal math)*/
+#     if [ ! -f ".${ROOT_SRC_DIR}/user_settings.h" ]; then
```

Comment:
Why is this commented out and still here?

Reply:
oh, that was an oversight. good catch, deleted.